index.bs

```diff
@@ -4130,6 +4283,31 @@ follows:</p>
 </ol>
 
 
+<h3 id="rules-for-parsing-attr-key-values-algorithm">WebVTT Attributes key/value Parsing Rules</h3>
+<p>The <dfn>WebVTT attributes key/value parsing rules</dfn> consist of the following algorithm.</p>
+
+<ol algorithm="WebVTT attributes block parsing">
+ <li>Let |input| be the list of key/value pairs from a <a>WebVTT attributes block</a>.</li>
+ <li>
+  How the attribute is processed depends on its key name, as follows:
+  <dl>
+
+   <dt>If the key name is "<code>kind</code>" (case-insensitive)</dt>
```

Comment:
You should define "(case-insensitive)" because it turns out that there are potential variations. Since the thing you're comparing to is an ASCII string, use Infra's "ASCII case insensitive" for best results.